htdocs/modules/publisher/admin/import/news.php

```diff
@@ -119,7 +119,7 @@
     $newArticleArray = array();
 
     $oldToNew = array();
-    while ($arrCat = $xoopsDB->fetchArray($resultCat)) {
+    while (false !== ($arrCat = $xoopsDB->fetchArray($resultCat))) {
 
```

Comment:
All new or changed lines in each commit are now being checked for compliance to PSR-2 during the travis-ci run. This empty line after the changed while statement is being flagged, and needs to be removed to pass:
```
FILE: htdocs/modules/publisher/admin/import/news.php
 122 | ERROR | [x] Blank line found at start of control structure
```